code/modules/client/preference_setup/origin/origin.dm

```diff
@@ -0,0 +1,249 @@
+/mob/living/carbon/human
+	var/decl/origin_item/culture/culture
+	var/decl/origin_item/origin/origin
+
+/datum/category_item/player_setup_item/origin
+	name = "Origin"
+	sort_order = 1
+
+/datum/category_item/player_setup_item/origin/load_character(var/savefile/S)
+	S["culture"]    		>> pref.culture
+	S["origin"]     		>> pref.origin
+	S["citizenship"]		>> pref.citizenship
+	S["religion"]			>> pref.religion
+	S["accent"]				>> pref.accent
+	S["economic_status"] 	>> pref.economic_status
+
+/datum/category_item/player_setup_item/origin/save_character(var/savefile/S)
+	S["culture"]			<< pref.culture
+	S["origin"]				<< pref.origin
+	S["citizenship"]		<< pref.citizenship
+	S["religion"]			<< pref.religion
+	S["accent"]				<< pref.accent
+	S["economic_status"]	<< pref.economic_status
+
+/datum/category_item/player_setup_item/origin/gather_load_query()
+	return list(
+		"ss13_characters" = list(
+			"vars" = list(
+				"culture",
+				"origin",
+				"economic_status",
+				"citizenship",
+				"religion",
+				"accent"
+			),
+			"args" = list("id")
+		)
+	)
+
+/datum/category_item/player_setup_item/origin/gather_load_parameters()
+	return list("id" = pref.current_character)
+
+/datum/category_item/player_setup_item/origin/gather_save_query()
+	return list(
+		"ss13_characters" = list(
+			"culture",
+			"origin",
+			"economic_status",
+			"citizenship",
+			"religion",
+			"accent",
+			"id" = 1,
+			"ckey" = 1
+		)
+	)
+
+/datum/category_item/player_setup_item/origin/gather_save_parameters()
+	return list(
+		"culture" = pref.culture,
+		"origin" = pref.origin,
+		"economic_status" = pref.economic_status,
+		"citizenship" = pref.citizenship,
+		"religion" = pref.religion,
+		"accent" = pref.accent,
+		"id" = pref.current_character,
+		"ckey" = PREF_CLIENT_CKEY
+	)
+
+//Maybe turn accents/religion/citizenship into decls?
+//Todo: remove species accent/citizenship lists.
+//Maybe todo: add a proc that returns the default/most common accent/religion/citizenship for a certain origin so we don't have to do list[1].
+/datum/category_item/player_setup_item/origin/sanitize_character(var/sql_load = 0)
+	var/datum/species/S = all_species[pref.species]
+	var/decl/origin_item/CI = S.get_default_culture()
```

Comment:
We don't need to get CI if the check on 75 is `FALSE` so we can move it under that if statement?